vignettes/cookbook_interpretation.Rmd

```diff
@@ -14,59 +14,164 @@ knitr::opts_chunk$set(
+
+If you use the `by_company = TRUE` option of the `target_sda()` function, most of what's said above is true except relative to the company listed in that row in the `name_abcd` column.
+
+For detailed information about the table output of the `target_sda()` function and its columns, see the [data dictionary](data_outputs.html#target_sda-output).
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+### Emission Intensity Plot
+
+The emission intensity metric measures the average CO^2^ intensity of the portfolio in the steel and cement sector. This emission intensity is given as CO^2^/economic unit of output (for example, CO^2^/per ton of steel produced). This is then compared to an emission intensity reference point set by a climate scenario.
+
+While this is not the main metric of choice for the largest sectors tackled in this methodology, the emission intensity of the activities financed by the portfolio is nonetheless the first metric in sectors for which no clear technology pathways have been set out (namely, steel and cement). Put differently, for these sectors, no zero-carbon alternative yet exists. As such, it is not possible to use the technology mix metric or the volume production volume trajectory metric to measure alignment. However, it is still imperative to steer capital in a way that aims to decrease carbon emissions in these sectors – hence the emission intensity metric is used.
+
+#### Explanation of Output
+
+[2 or 3 sentence explanation on plot output]
+
+#### Interpretation & Intuition
+
+[2 or 3 sentences about interpretation]
+
+[2 or 3 sentences about intuition]
+
+```{r, message=FALSE, warning=FALSE, echo=FALSE}
+library(pacta.loanbook)
+
+sda_targets <-
+  target_sda(
+    data = prioritize(match_name(loanbook_demo, abcd_demo)),
+    abcd = abcd_demo,
+    co2_intensity_scenario = co2_intensity_scenario_demo,
+    region_isos = region_isos_demo
+  )
+
+qplot_emission_intensity(
+  dplyr::filter(
+    sda_targets,
+    sector == "cement",
```

Comment:
```suggestion
    sector == "steel",
```

For whatever reason, the steel demo data seems to have more space between each line, making it a bit easier to interprate.